airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py

```diff
@@ -885,15 +888,7 @@ def _validate_expected_records(
         actual_by_stream = self.group_by_stream(records)
         for stream_name, expected in expected_records_by_stream.items():
             actual = actual_by_stream.get(stream_name, [])
-            detailed_logger.info(f"Actual records for stream {stream_name}:")
```

Comment:
I'm still curious as if we should still print the actual so that user can just copy/paste it to the `expected_records.jsonl`

Reply:
The purpose of leaving it would be to make it easier to update the expected records? If so leaving it in would make sense to me assuming people typically use this output for that. I'm not familiar enough with development involving CAT updates to say for sure though.